render.ts

```diff
@@ -11,13 +11,30 @@ import renderToString from 'https://esm.sh/preact-render-to-string@6.1.0?deps=pr
 import { VNode } from 'https://esm.sh/preact@10.17.1?target=es2022'
 import { Context } from './mod.ts'
 
+const HEAD_TAG = '<head>'
+
 export function render(c: Context, Component: VNode) {
   const htmlString = renderToString(Component)
 
   try {
     const { html, css } = extract(htmlString)
-
-    c.res.body = `<style>${css}</style><body>${html}</body>`
+    console.log(css.length)
+    const startIdxOfHeadContent = html.indexOf(HEAD_TAG)
+    const endIdxOfHeadContent = html.indexOf(
+      `${HEAD_TAG.at(0)}/${HEAD_TAG.slice(1)}`,
```

Comment:
why not just use `</head>`? :p

Reply:
that's true, I go a bit over the top sometimes lol...

Reply:
Pushed changes for this.